editor/src/components/canvas/ui-jsx-canvas.spec.tsx

```diff
@@ -1405,8 +1405,12 @@ export var ${BakedInStoryboardVariableName} = (props) => {
             \\"
             data-uid=\\"scene-aaa\\"
           >
-            <div id=\\"nasty-div\\" data-uid=\\"ce9~~~1\\" data-path=\\"d1b~~~2/ce9~~~1\\">
-              huhahuha
+            <div
+              id=\\"nasty-div\\"
+              data-uid=\\"aaa\\"
+              data-path=\\"utopia-storyboard-uid/scene-aaa/app-entity:aaa\\"
+            >
+              huha huha
```

Comment:
Oh my, why was this previously getting rendered without a space?!

Reply:
Good question.